lms/djangoapps/course_blocks/transformers/library_content.py

```diff
@@ -177,3 +188,68 @@ def publish_event(event_name, result, **kwargs):
             format_block_keys,
             publish_event,
         )
+
+
+class ContentLibraryOrderTransformer(BlockStructureTransformer):
+    """
+    A transformer that manipulates the block structure by modifying the order of the
+    selected blocks within a library_content module to match the order of the selections
+    made by the ContentLibraryTransformer or the corresponding XBlock. So this transformer
+    requires the selections for the randomized content block to be already
+    made either by the ContentLibraryTransformer or the XBlock.
+
+    Staff users are *not* exempted from library content pathways/
```

Comment:
Nit: I'm not sure what "pathways" means here nor why it ends with a `/`

Reply:
@bradenmacdonald, the traling `/` is a typo and I will fix it. I have just copied the text from the `ContentLibraryTransformer` [docstring](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/course_blocks/transformers/library_content.py#L29) since it also applies to the related `ContentLibraryOrderTransformer.

Reply:
I have fixed the typo.